src/main/window-manager.ts

```diff
@@ -7,7 +7,7 @@ import { initMenu } from "./menu";
 import { tracker } from "../common/tracker";
 
 export class WindowManager {
-  protected mainView: BrowserWindow;
+  public readonly mainView: BrowserWindow;
```

Comment:
Is WindowManager exported to extensions api? Just wondering if it is better in terms of api hardening to add a getter for `mainView` instead.

Reply:
The generated JS (even for a getter) is no different. As is easily seen by what we do in some of our tests by casting to any (or just by using JS instead of TS).

If we really wanted to be hardened we would use the new JS `#name` private identifiers. But I don't know if we want to do that.

Reply:
So it won't break a TS extension if one day we needed to change `mainView` field back to being `protected`?

Reply:
I don't understand, if we have `protected _mainView` and a `public get mainView()`. Then that is completely identical to `public readonly`. Modulo what the field names are.